### ISSUE TITLE
Feature request: chatting with pdf file

### ISSUE THREAD

**Issue by softpedia-inc** [Mobile-Artificial-Intelligence/maid#601]
Dear sir, when can we enjoy the possibility of chatting with a pdf book or pdf document completely offline? Other apps have this option but they must be connected to the internet.

**Comment by danemadsen** [comment]
are you talking about RAG?

**Comment by softpedia-inc** [comment]
> are you talking about RAG?

Yes sir/madam, this feature is being incorporated in newer ai chat apps as it allows for precise answers related to a pdf file or instructions document.
For example the user could be searching for how to install a certain driver from a large pdf manual and with this feature he could retrieve the exact information needed without going through the whole document.
